includes/admin/class-wc-stripe-admin-notices.php

```diff
@@ -512,6 +513,49 @@ public function subscription_check_detachment() {
 		}
 	}
 
+	/**
+	 * Add a notice to the admin area if there are subscriptions with payment method detached.
+	 *
+	 * @return void
+	 */
+	public function subscription_check_detachment_bulk_action() {
+		if ( ! empty( $_REQUEST['detached-subscriptions'] ) ) {
+			$detached_subs_ids = explode( ',', wp_unslash( $_REQUEST['detached-subscriptions'] ) ); // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+			$subscriptions     = [];
+			foreach ( $detached_subs_ids as $detached_sub_id ) {
+				$detached_sub_id = absint( $detached_sub_id );
+				$subscription    = wcs_get_subscription( $detached_sub_id );
+				if ( ! $subscription instanceof WC_Subscription ) {
+					continue;
+				}
+				$subscriptions[] = WC_Stripe_Subscriptions_Helper::get_detached_data_from_subscription( $subscription );
+			}
+			$detached_messages = WC_Stripe_Subscriptions_Helper::build_subscriptions_detached_messages( $subscriptions );
+
+			$notice_content = '';
+			if ( empty( $detached_messages ) ) {
+				$notice_content .= '<p>' . esc_html__( 'No detached subscriptions found.', 'woocommerce-gateway-stripe' ) . '</p>';
```

Comment:
I do not get this notice when I select and check subscriptions that have attached payment methods.